src/pages/components/Overlay.qml

```diff
@@ -523,6 +524,7 @@ Background {
                 model: webView.tabModel
                 portrait: tabPage.isPortrait
                 privateMode: webView.privateMode
+                pullDownVisible: webView.visibility >= QuickWindow.Window.Maximized
```

Comment:
A little curious why this is done. Tab page per se not included in cover? Introduced 2014 with commit "Hide pulley menu, close tab button, title, and the opacity ramp in cover"

Reply:
Ah, so true :+1: Will cleanup.